instrumentation/active_storage/lib/opentelemetry/instrumentation/active_storage/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module ActiveStorage
+      VERSION = '0.0.0'
```

Comment:
let's make it '0.1.0' as initial release version.

Reply:
changed the version in https://github.com/open-telemetry/opentelemetry-ruby-contrib/commit/d6979a7eff054d8e7f2c67dd62e0929eff4b3554